models/ingredient.js

```diff
@@ -0,0 +1,46 @@
+
+const {Schema, model} = require('mongoose');
+const {handleMongooseError} = require("../helpers");
+
+const ingredientSchema = new Schema({
+
+title:{
+    type: String,
+},
+ingredientThumb:{
+    type: String,
+},
+abv:{
+    type: String,
+},
+
+alcohol:{
+    type: String,
+},
+
+description:{
+    type: String,
+},
+
+type:{
+    type: String,
+},
+
+flavour:{
+    type: String,
+},
+
+country:{
+    type: String,
+},
+}
+);
+
+
+ingredientSchema.post('save', handleMongooseError)
```

Comment:
Are we going to make POST request for ingredients? If yes, please add JoiShema and make appropriative endpoint.

Reply:
amended in last commit